### ISSUE TITLE
[jobs/police] Weapon attachments for the Carbine Rifle

### ISSUE THREAD

**Issue by sahara87** [InZidiuZ/op-framework-issue-tracker#810]
**Name:**
Kiwi#6887

**Summary:**
Weapon attachments for the Carbine Rifle: Grip, Flashlight and Scope.

**Reason:**
Realistically, a PD Carbine Rifle would be equipped with attachments to give the officer using it, an advantage. I think the grip should improve the stability/recoil of the weapon, as a police officer would be specially trained to use the weapon, meaning they would be able to control it a lot better than we can now. A flashlight would be helpful when dealing with situations in the dark or having to breach buildings or houses and would also allow people with darker texture packs to see their surroundings in a breach situation. The scope would allow us to have better accuracy with the weapon, meaning it would be a lot better in a long-range gun fight. A scope would just be an overall helpful and useful attachment to have on this weapon.

**Comment by Imaghost1337** [comment]
Might as well give police rocket launchers.

The amount of things that makes cop life easier is funny to me.
Usually there's more then 15 of you in late hours after 2nd restart
You got police cruisers, when even the slowest goes 190mph, which is quicker then 90% of all cars
You got helicopters, with cameras that see through walls
You recently got the carbine rifles

 Not hating here, just the amount of things that is given to police is funny, when a huuuge majority of criminals runs around with pistols

**Comment by joevapes3** [comment]
The idea that was passed through after we used it in SWAT training was that we wanted to get a flashlight attachment. I personally don't feel we need anything else beyond that for it at this time. With the several times I've tested it, the recoil is definitely strong, but not unmanageable with proper firing techniques, and spray control. The flashlight is a reasonable request. Anything beyond that is excessive imo.
